browser/ui/omnibox/brave_omnibox_client_impl.cc

```diff
@@ -118,6 +131,9 @@ void BraveOmniboxClientImpl::OnAutocompleteAccept(
     if (search_engine_tracker_ != nullptr) {
       search_engine_tracker_->RecordLocationBarQuery();
     }
+#if BUILDFLAG(ENABLE_AI_CHAT)
+    ai_chat_metrics_->RecordOmniboxSearchQuery();
+#endif
```

Comment:
Just double checking, this seems to be called for all search query, not only "Leo" related match. Is it what we want?

Reply:
yep, totally fine. we use this to compare the amount of Ask Leo queries vs. normal search queries

Reply:
Aha! Thanks for explaining 👍